charts/opentelemetry-collector/values.schema.json

```diff
@@ -135,6 +135,10 @@
         "existingName": {
           "description": "Specifies an existing configMap to be mounted to the pod",
           "type": "string"
+        },
+        "configTemplate": {
```

Comment:
This should be renamed to `existingPath`

Reply:
Good catch.